analyzer/src/main/kotlin/managers/utils/NuGetSupport.kt

```diff
@@ -126,8 +126,25 @@ class NuGetSupport {
     private fun getAllPackageData(id: Identifier): NuGetAllPackageData {
         // Note: The package name in the URL is case-sensitive and must be lower-case!
         val lowerId = id.name.toLowerCase()
-        val dataUrl = "$registrationsBaseUrl/$lowerId/${id.version}.json"
-        val data = runBlocking { mapFromUrl<PackageData>(JSON_MAPPER, dataUrl) }
+
+        var data: PackageData? = null
+        for (baseUrl in registrationsBaseUrls) {
+            val dataUrl = "$baseUrl/$lowerId/${id.version}.json"
+            data = try {
+                runBlocking { mapFromUrl<PackageData>(JSON_MAPPER, dataUrl) }
+            } catch (e: IOException) {
+                log.debug("Failed to retrieve package from $baseUrl")
```

Comment:
Please always use the curly-braces syntax for `log`, so the string only gets interpolated if the respective logging level is active, i.e. use `log.debug { "Failed to retrieve package from $baseUrl." }` (and add a dot at the end of the log sentence 😉 ).